include/af/lapack.h

```diff
@@ -200,6 +200,26 @@ namespace af
     */
     AFAPI array inverse(const array &in, const matProp options = AF_MAT_NONE);
 
+#if AF_API_VERSION >= 37
+    /**
+       C++ Interface for pseudo-inverting (Moore-Penrose) a matrix.
+       Currently uses the SVD-based approach.
+
+       \param[in] in is the input matrix
+       \param[in] tol defines the lower threshold for singular values from SVD
+       \param[in] options determining various properties of matrix \p in
+       \returns \p x, the inverse of the input matrix
+
+       \note \p tol is not the actual lower threshold, but it is passed in as a parameter to the calculation of the actual threshold relative to the shape and contents of \p in.
```

Comment:
80 characters per line. You can tab to the first character of the comment to make it more readable:

```

 \note \p tol is not the actual lower threshold, but it is passed in as a
       parameter to the calculation of the actual threshold relative to the
       shape and contents of \p in.
```

Reply:
Same case as above for this file, just added line breaks now (to pinverse and af_pinverse)